Elf/__init__.py

```diff
@@ -231,9 +231,11 @@ def _parseSections(self):
             if secslen != len(secbytes):
                 logger.warning('Invalid Section-Headers Size: should be: %d   retrieved: %d', secslen, len(secbytes))
 
-            vstruct.VArray(elems=secs).vsParse(secbytes, fast=True)
+            secs = vstruct.VArray()
```

Comment:
if this is the direction, be sure to clean up the creation of `secs` above.  if you don't need it kill `sec` as well